### DISCUSSION TITLE
Custom Modal Content

### DISCUSSION THREAD

**Discussion by ramiropiro** [filamentphp/filament#5528] [General]
hello, how can I pass a actual record from a table to a custom modal view, and how do I get the parameter in the blade view? Thanks.

Table
![image](https://user-images.githubusercontent.com/111913304/214140072-bdf077ed-6593-497b-9d99-4a25cac32043.png)

Action on the table
![image](https://user-images.githubusercontent.com/111913304/214140143-f5cea02e-d894-4827-953b-58fa2c053f47.png)

custom Blade view
![image](https://user-images.githubusercontent.com/111913304/214140248-ee0a7bc0-f4ae-438a-ab2c-bc896a537d01.png)

**Comment by danharrin** [comment]
`modalContent(fn ($record) => view('xxx', ['record' => $record]))`, then you can access $record in your view and anything else you pass in

**Reply by ramiropiro** [reply]
Thank you

**Reply by casic** [reply]
And if i am in  relation manager ???  not work  ??

**Reply by Rattone** [reply]
> `modalContent(fn ($record) => view('xxx', ['record' => $record]))`, then you can access $record in your view and anything else you pass in

How to close a modal opened like this after, for example, an action in the modal?

**Reply by fikurimax** [reply]
> > `modalContent(fn ($record) => view('xxx', ['record' => $record]))`, then you can access $record in your view and anything else you pass in
> 
> How to close a modal opened like this after, for example, an action in the modal?

Have you got a way?

**Comment by elmergustavo** [comment]
perfect❤️
